### PR TITLE
Use ColorProperty for remaining color properites

### PR DESCRIPTION
**Pull request by pythonic64**
Continuation of https://github.com/kivy/kivy/pull/6918

Use ColorProperty for:
- GestureSurface.color
- Label.outline_color
- Label.disabled_outline_color

### CODE REVIEW

File: kivy/uix/gesturesurface.py [hunk 1]
```diff
@@ -238,9 +238,13 @@ class GestureSurface(FloatLayout):
             Color used to draw the gesture, in RGB. This option does not
             have an effect if :attr:`use_random_color` is True.
 
-            :attr:`draw_timeout` is a
+            :attr:`color` is a
             :class:`~kivy.properties.ListProperty` and defaults to
```
Comment (tshirtman) [inline kivy/uix/gesturesurface.py:242]:
```suggestion
            :class:`~kivy.properties.ColorProperty` and defaults to
```

File: kivy/uix/label.py [hunk 2]
```diff
@@ -792,10 +792,15 @@ def on_ref_press(self, ref):
     .. versionadded:: 1.10.0
 
     :attr:`outline_color` is a :class:`~kivy.properties.ListProperty` and
```
Comment (tshirtman) [inline kivy/uix/label.py:794]:
```suggestion
    :attr:`outline_color` is a :class:`~kivy.properties.ColorProperty` and
```

File: kivy/uix/label.py [hunk 3]
```diff
@@ -806,6 +811,11 @@ def on_ref_press(self, ref):
 
     :attr:`disabled_outline_color` is a :class:`~kivy.properties.ListProperty`
```
Comment (tshirtman) [inline kivy/uix/label.py:812]:
```suggestion
    :attr:`disabled_outline_color` is a :class:`~kivy.properties.ColorProperty`
```

File: kivy/uix/gesturesurface.py (hunk 1 above)
Comment (pythonic64) [reply kivy/uix/gesturesurface.py:242]: The existing approach is the same as in previous pull request. Like this for example: https://github.com/kivy/kivy/blob/master/kivy/uix/actionbar.py#L214, and that was approved by matham.

Comment (matham) [reply kivy/uix/gesturesurface.py:242]: I think I overlooked that by only looking at the diff 😬 sorry

Comment (tshirtman) [reply kivy/uix/gesturesurface.py:242]: mistakes have been made :P

**Pull request merged**